go/runtime/enclaverpc/api/grpc.go

```diff
@@ -53,23 +52,59 @@ func handlerCallEnclave( // nolint: golint
 }
 
 // RegisterService registers a new EnclaveRPC transport service with the given gRPC server.
-func RegisterService(server *grpc.Server, service Transport) {
-	server.RegisterService(&serviceDesc, service)
+func (e *EnclaveRPC) RegisterService(server *grpc.Server, service Transport) {
+	server.RegisterService(&e.ServiceDesc, service)
+}
+
+// EnclaveRPC creates a new EnclaveRPC service.
+func New(serviceNamePrefix string, accessControl func(req interface{}) bool) *EnclaveRPC {
```

Comment:
Maybe call it `NewService` or `NewGRPCService` as `New` seems to generic (there are other structs in this package).